src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1080,7 +1080,7 @@
   - name: --settings
     short-summary: Space-separated appsettings in KEY=VALUE format. Use @{file} to load from a file.
   - name: --slot-settings
-    short-summary: Space-separated slot appsettings in KEY=VALUE format. Use @{file} to load from a file.
+    short-summary: Space-separated appsettings in KEY=VALUE format. Use @{file} to load from a file. Given setting are added to the configuration and marked as Deployment slot setting by default.
```

Comment:
Is this supposed to be used only to mark app settings as Deployment slot settings? if so I would modify this as "
App settings that should be enabled as deployment slot settings. The app settings can be represented as 
space-separated  KEY=VALUE pair or Use @{file} to load from a file."

Reply:
Did we test what happens if a user uses --settings with JSON file that has "slot settings: true" like portal client suggests? Does the command fail ?

Reply:
Modified the help text and tested as suggested. Found no issues.